src/org/omegat/filters2/master/PluginUtils.java

```diff
@@ -96,6 +96,8 @@ public enum PluginType {
         DICTIONARY("dictionary"),
         /** theme, that register Swing Look-and-Feel with OmegaT properties into UIManager. */
         THEME("theme"),
+        /** team repository version control system connector plugins. */
+        TEAMREPOSITORY("teamRepository"),
```

Comment:
It is better to be  simple "repository", with comment "remote repository connector plugins."